packages/relayer-lib/src/tx_builder/cosmos_to_cosmos.rs

```diff
@@ -137,6 +138,14 @@ impl TxBuilderService<CosmosSdk, CosmosSdk> for TxBuilder {
             .chain(recv_msgs.into_iter().map(|m| Any::from_msg(&m)))
             .chain(ack_msgs.into_iter().map(|m| Any::from_msg(&m)))
             .collect::<Result<Vec<_>, _>>()?;
+        if all_msgs.len() == 1 {
```

Comment:
It's not immediately clear why len == 1 means no messages to relay to Cosmos? Maybe a small comment to explain that?